components/lists/List.js

```diff
@@ -0,0 +1,49 @@
+import styled from "styled-components";
+import dynamic from "next/dynamic";
+import { useListsStore } from "../../useStore";
+import { DragDropContext, Droppable } from "@hello-pangea/dnd";
+
+//Components
+const DynamicEntry = dynamic(() => import("./Entry"), {
+  ssr: false,
+});
```

Comment:
Why are you using next/dynamic here? The Entry component can render server side, right?

Reply:
Nope. It would throw an error, if you add new Entries (at least local)